src/components/title/create/title-create.js

```diff
@@ -49,22 +56,24 @@ export default class TitleCreate extends Component {
   createFormRef = createRef();
 
   renderFirstMenu = () => {
-    const { onCancel } = this.props;
+    const {
+      onCancel,
+      intl,
+    } = this.props;
+
+    const label = intl.formatMessage({
+      id: 'ui-eholdings.label.icon.closeX',
+    }, {
+      paneTitle,
+    });
```

Comment:
This is a bit of a useless refactoring. To remove `<FormattedMessage />` wrapper you add `injectIntl` with a new prop for this component. It's not a good idea. In this case better to leave it as it or re write component to functional.